build_and_deploy_container.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -e
+set -x
+
+IMAGE_TAG=${1:?Please provide a tag for the image}
```

Comment:
I suggest using the `USAGE=` trick like the other deploy scripts for giving a hint about the other parameters if we forget the first.